vars/RedHat_10.yml

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: MIT
+---
+# Put internal variables here with Red Hat Enterprise Linux 9 specific values.
+
+# List of repositories holding HA cluster packages.
+# id: repo ID used to enable the repo
+# name: user-friendly name of a repo used to check if the repo is enabled
+__ha_cluster_repos:
+  - id: rhel-9-for-{{ ansible_architecture }}-highavailability-rpms
+    name: High Availability
+  - id: rhel-9-for-{{ ansible_architecture }}-resilientstorage-rpms
```

Comment:
Is 'rhel-9' in RHEL 10 config a mistake or a temporary solution? When to switch to rhel-10?

Reply:
Automation copied vars/RedHat_9.yml into vars/RedHat_10.yml. Can you check if repository for rhel 10 exists?

Reply:
Changed to 10

Reply:
Right now, rhel-10-for-x86_64-highavailability-beta-rpms and rhel-10-for-x86_64-resilientstorage-beta-rpms exist. I think the role must identify whether beta version is used by looking into /etc/redhat-release and set repos based on that. I'll do this in a separate PR.

Reply:
@richm does it make sense to support beta? Or we can hardcode GA values that will work with RHEL 10 ga?

Reply:
> @richm does it make sense to support beta? Or we can hardcode GA values that will work with RHEL 10 ga?

Not sure.  Whatever we do, it should work for

* tox ... -- --image-name rhel-10-beta
* tox ... -- --image-name rhel-10

and also for downstream testing.  Note that in all of these cases, the image/managed node is already configured with the appropriate repos, so we need to ensure that the ha_cluster tasks that enable these repos are skipped.